addons/sale_project/models/project_project.py

```diff
@@ -405,13 +439,22 @@ def _get_sale_items_domain(self, additional_domain=None):
             domain = expression.AND([domain, additional_domain])
         return domain
 
-    def _get_sale_items(self, with_action=True):
-        domain = self._get_sale_items_domain()
+    def _get_items_id_per_section_id(self):
         return {
-            'total': self.env['sale.order.line'].sudo().search_count(domain),
-            'data': self.get_sale_items_data(domain, limit=5, with_action=with_action),
+            'materials': [],
+            'service_revenues': [],
         }
 
+    def _get_domain_from_section_id(self, section_id):
+        if section_id == 'materials':
+            return self._get_sale_items_domain([('product_id.type', '!=', 'service')])
+        #  When the sale_timesheet module is not installed, all service products are grouped under the 'service revenues' section.
+        return self._get_sale_items_domain([('product_id.type', '=', 'service')])
```

Comment:
Maybe, if could be easier to write it in one line and just change the operator. 🤷🏼‍♂️ 
Also, we could use `product_type` instead of `product_type`. 🙂 

```suggestion
        #  When the sale_timesheet module is not installed, all service products are grouped under the 'service revenues' section.
        return self._get_sale_items_domain([('product_type', '!=' if section_id == 'materials' else '=', 'service')])
```

🤷🏼‍♂️ 

It is up to you for the first suggestion (doing a ternary operator to just change the operator to use inside the domain). 🙂